src/app/recommendation/create-recommendation/create-recommendation.component.html

```diff
@@ -85,7 +85,7 @@
     >
       Save draft
     </button>
-    <button
+    <button *ngIf="!isRevalAdmin"
```

Comment:
Just wondering based on the requirements should it instead be ngIf="isRevalApprover" (and replace the bit in the service.ts with a includes("RevalApprover") - just because it's an exclusive feature for that role (Observers shouldn't be able to see this button either right?)

Reply:
Thanks @CaiWillis . Nice suggestion. I have mage changes accordingly